docs/source/rclone.rst

```diff
@@ -0,0 +1,420 @@
+=======================================
+Awscli/Rclone Submission (experimental)
+=======================================
+
+For some use cases we are trialing a more streamlined submission process that does
+not involve the installation of Python or SubmitCGAP and instead involves transferring
+files directly into a `PUBLIC-WRITE` S3 bucket defined by us. The idea is you will source
+credentials associated with the datastore where your files are kept and sync them
+directly into the submission bucket, along with the submission Excel (for data
+accessioning).
+
+These instructions are intended to support both `awscli` if transferring directly from
+`AWS S3` or `rclone` if you are transferring from other Cloud Providers (still compatible
+with `AWS S3`). At this time we recommend using `rclone`, since it is compatible with
+most popular Cloud Providers.
+
+
+Installing rclone
+-----------------
+
+Navigate to the `rclone` `download page <https://rclone.org/downloads/>`_ and click the
+download button that corresponds to the operating system you are running. Once installed
+you should see the `rclone` command in your path.
+
+.. code-block:: bash
+
+    $ which rclone
+
+
+Configuring Anonymous S3 Remote
+-------------------------------
+
+The `rclone` `S3 documentation page <https://rclone.org/s3/>`_ has detailed information
+on how to configure S3 remotes. If your submission files are not on AWS S3, you will need
+to create an anonymous S3 remote for use with the transfer. The bucket name we will give
+you will allow anyone to submit files to it but will only us internally to read the files.
+
+Upon running `rclone config` you should be greeted with detailed setup instructions that
+the below block emulates to walk you through the process. Our submission bucket will be
+in us-east-1, but be aware if transferring from another AZ on S3 you will incur S3
+egress charges when transferring into our bucket.
+
+If you are submitting from S3, you will want to follow very similar configuration as
+the below except that you will want to pass your account access key ID and secret so
+you can access the files in your account. You will not need access credentials to submit
+to our buckets, though that can be arranged if it is preferred.
+
+
+.. code-block:: bash
+
+    $ rclone configure
+    No remotes found, make a new one?
+    n) New remote
+    s) Set configuration password
+    q) Quit config
+    n/s/q> n
+    name> cgap_s3_submission
+    Type of storage to configure.
+    Choose a number from below, or type in your own value
+    [snip]
+    XX / Amazon S3 Compliant Storage Providers including AWS, Ceph, ChinaMobile, ArvanCloud, Dreamhost, IBM COS, Minio, and Tencent COS
```

Comment:
Is the "XX" here just an actual artifact in the typeout or is it some meta-notation?